sub_circuit_identification/src/write_verilog_lef.py

```diff
@@ -280,6 +362,8 @@ def generate_lef(fp, name, values, available_block_lef,
     # write a verilog file
     VERILOG_FP = open(RESULT_DIR + INPUT_PICKLE + '.v', 'w')
     LEF_FP = open(RESULT_DIR + INPUT_PICKLE + '_lef.sh', 'w')
+    logging.info("writing spice file for cell generator")
+    SP_FP = open(RESULT_DIR + INPUT_PICKLE + '_blocks.sp', 'w')
```

Comment:
You are using names like NMOS_VTL instead of NMOS here. Can we add some .MODEL statements if you are still have the map somewhere? Or is sub_circuit_identification actually only matching for NMOS_VTL right now for DP_*MOS & CMC_*MOS

Reply:
I don't have any mapping to "NMOS" ( are these xyce standards?). I am using the type of the device written in basic_template  I will modify the basic template file to NMOS which can solve this.

Reply:
Let's figure out what we are using to simulate the schematic. And if we have a working simulation in Xyce, we should be reusing the model definitions we used to get the simulation going. @meghna09 can you comment?